### PR TITLE
Reset netsh proxy

### PR DESCRIPTION
**Pull request by joinimran**
As user requested, I have added netsh reset command for reference as well as attached the help doc to get more details related to netsh.

Problem: https://github.com/MicrosoftDocs/windows-itpro-docs/issues/3372

### CODE REVIEW

**Comment by mypil** [comment]
@nenonix @JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: windows/security/threat-protection/windows-defender-atp/configure-proxy-internet-windows-defender-advanced-threat-protection.md [hunk 1]
```diff
@@ -87,6 +87,12 @@ netsh winhttp set proxy <proxy>:<port>
 ```
 For example: netsh winhttp set proxy 10.0.0.6:8080
 
+To reset the winhttp proxy, enter the following command and press **Enter**
+```
+netsh winhttp reset proxy
+```
+To get more details how to use netsh command,[please see help doc](https://docs.microsoft.com/en-us/windows-server/networking/technologies/netsh/netsh-contexts).
```
Comment (nenonix) [inline windows/security/threat-protection/windows-defender-atp/configure-proxy-internet-windows-defender-advanced-threat-protection.md]:
```suggestion
See [Netsh Command Syntax, Contexts, and Formatting](https://docs.microsoft.com/windows-server/networking/technologies/netsh/netsh-contexts) to learn more.
```

**Comment by joinimran** [comment]
Suggested changes applied.

**Comment by mypil** [comment]
@mjcaparas - Can you please assist to check, approve and merge the content for this PR based on issue #3372?

Thank you.

**Pull request merged**